src/applications/simple-forms/21-10210/tests/e2e/fixtures/mocks/feature-toggles.json

```diff
@@ -0,0 +1,6 @@
+{
+  "data": {
+    "type": "feature_toggles",
+    "features": []
+  }
+}
```

Comment:
We have a shared feature-toggles we could use instead. It enables 26-4555 but should not affect this form. I figure we can enable every form that has a flipper in there as needed so tests run.

Reply:
Now importing shared file.